### PR TITLE
test: Updated repo for Travis/Jenkins CI usage through tox.

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

**Comment by ashcrow** [comment]
@cooktheryan PTAL

**Comment by ashcrow** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;hAVQYymt&#x2F;468-2-multi-host-mgr-2-readd-repos-to-travis"> (2) [multi-host-mgr-2] Readd repos to Travis</a>

File: tox.ini [hunk 1]
```diff
@@ -7,7 +7,7 @@ deps =
     -rrequirements.txt
     -rtest-requirements.txt
 
-[testenv:py35]
+[testenv:py27]
```
Comment (mbarnes) [inline tox.ini]: Why this change?

Comment (ashcrow) [inline tox.ini]: Oops.

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by mbarnes** [comment]
Hmm, does the "squash and merge" button do what I expect?  Let's find out...

**Comment by mbarnes** [comment]
It does, neat!
